data/mods/Magiclysm/Spells/item_only.json

```diff
@@ -103,5 +103,39 @@
     "spell_class": "NONE",
     "message": "Your veins feel like they are on fire!\nYou start regenerating mana!",
     "extra_effects": [ { "id": "potion_recover_mana" } ]
+  },
+  {
+    "id": "lupercalia",
+    "type": "SPELL",
+    "name": "Lupercalian Release",
+    "//": "Used for wolfshead cufflinks not castable",
+    "description": "This spell transforms you into a werewolf.  There may be some cons to this.",
+    "valid_targets": [ "self" ],
+    "flags": [ "SILENT", "NO_LEGS", "NO_HANDS" ],
+    "effect": "spawn_item",
+    "effect_str": "aura_werewolf",
+    "max_level": 1,
+    "min_damage": 1,
+    "max_damage": 1,
+    "damage_increment": 0,
+    "damage_type": "biological",
+    "difficulty": 0,
+    "spell_class": "NONE",
+    "base_casting_time": 300,
+    "base_energy_cost": 2,
+    "min_duration": 43200,
```

Comment:
432 seconds isn't a very long time. in fact, it's unlikely you'll get the healing intermittent activation.